polygon-digital-carbon/subgraph.template.yaml

```diff
@@ -622,3 +621,17 @@ templates:
         - event: ExAnteMinted(indexed uint256,indexed uint256,indexed address,uint256)
           handler: handleExAnteMinted
       file: ./src/TransferHandler.ts
+  - name: C3RetirementMetadata
+    kind: file/ipfs
+    mapping:
+      apiVersion: 0.0.7
+      language: wasm/assemblyscript
+      file: ./src/MetadataHandler.ts
+      handler: handleC3RetirementMetadata
+      entities:
+        - C3RetireRequest      
+        - C3RetirementMetadata
+        - C3MetadataProject
+      abis:
+        - name: ICRProjectToken
+          file: ../lib/abis/ICRProjectToken.json
```

Comment:
is there a reason we're using the ICR ABI with a supposedly C3 project token?

Reply:
ipfs file datasources require an abi here as a placeholder but it doesn't matter which one as it is never used:

https://thegraph.com/docs/en/developing/creating-a-subgraph/#known-issues